package.json

```diff
@@ -86,5 +78,8 @@
   "overrides": {
     "flat": ">=5.0.1",
     "minimatch": ">=3.0.5"
+  },
+  "browser": {
+    "fs": false
```

Comment:
This prevents bundler for containing incompatible `fs` lib to browser enviroments.